main.py

```diff
@@ -1,7 +1,44 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
 def main() -> QuerySet:
-    pass
+
+    # Create
```

Comment:
Remove these comments, there is no need for them. The code is self-explicit enough